### DISCUSSION TITLE
Astro format

### DISCUSSION THREAD

**Discussion by millette** [biomejs/biome#913] [General]
Glad to hear about biome vs prettier!

I see your website is built with astro but looking at its package.json it's not obvious if biome supports <https://github.com/withastro/prettier-plugin-astro>.

**Comment by ematipico** [comment]
We have a section where we state the languages that we support: https://biomejs.dev/internals/language-support

Astro is not one of them

**Comment by minht11** [comment]
Is there a roadmap or more information if or when new languages gonna be supported? My team is pending vue/astro support, before we could switch to Biome.

**Reply by ematipico** [reply]
You should expect a roadmap soon, although remember that we are a small team and works will be relatively slow unless we get more help

**Reply by angel-luis** [reply]
It would be nice to see Astro in Biome!
